### PR TITLE
[NFC] dev/core#1466 Update Documentation URLS to be the correct links…

### PR DESCRIPTION
**Pull request by seamuslee001**
… in the security component check

Overview
----------------------------------------
Updates the documentation links from pointing at the Wiki to be pointing at the mkdocs

ping @MikeyMJCO @eileenmcnaughton

### CODE REVIEW

File: CRM/Utils/Check/Component/Security.php [hunk 1]
```diff
@@ -364,7 +364,7 @@ public function isDirAccessible($dir, $url) {
    * @return string
    */
   public function createDocUrl($topic) {
-    return CRM_Utils_System::getWikiBaseURL() . $topic;
+    return CRM_Utils_System::docURL2('sysadmin/setup/security#' . $topic, TRUE);
```
Comment (MikeyMJCO) [inline CRM/Utils/Check/Component/Security.php]:
```suggestion
    return CRM_Utils_System::docURL2('sysadmin/setup/security#' . $topic, TRUE);
```
where is "docurl2" defined as a function?

Comment (seamuslee001) [reply CRM/Utils/Check/Component/Security.php]: @MikeyMJCO https://github.com/civicrm/civicrm-core/blob/master/CRM/Utils/System.php#L1288

File: CRM/Utils/Check/Component/Security.php [hunk 2]
```diff
@@ -71,7 +71,7 @@ public function checkLogFileIsNotAccessible() {
           $url[] = $log_path[1];
           $log_url = implode($filePathMarker, $url);
           if ($this->fileExists($log_url)) {
-            $docs_url = $this->createDocUrl('checkLogFileIsNotAccessible');
```
Comment (eileenmcnaughton) [inline CRM/Utils/Check/Component/Security.php]: If I enter https://wiki.civicrm.org/confluence/display/CRMDOC//uploads-should-not-be-accessible I get a 404 - unchanged I get redirected to the security page on the docs - ie the equivalent of

return CRM_Utils_System::docURL2('sysadmin/setup/security#' . $topic, TRUE);

So this change make it un-better

Comment (seamuslee001) [reply CRM/Utils/Check/Component/Security.php]: isn't that what L367 is meant to be doing?)

**Comment by jaapjansma** [comment]
Betty and I are reviewing PR's  and we came across this PR. 

We are unable to reproduce this PR as it is not clearly explained. However we do think this PR is ready to be merged as it is only updating non-working links to the CiviCRM documentation. So not a risk of breaking anything. And if it breaks then it breaks existing broken links. 

@eileenmcnaughton or @mattwire could one of you merge this PR?

**Comment by mattwire** [comment]
Merging based on @jaapjansma review @seamuslee001 please follow up with a subsequent PR if there are still broken links (per @eileenmcnaughton comment above)

**Comment by MikeyMJCO** [comment]
**Note:** We can refactor the underlying system here next year - we'll be adding metadata (catogorisation/tags) to docs (2020) and an API to pull docs information in UI along with an extension using it planned for 2021.

**Pull request merged**